lib/shopify_api/only_send_mutated_params.rb

```diff
@@ -0,0 +1,34 @@
+module ShopifyAPI
+  module OnlySendMutatedParams
+    def self.included(base)
+      base.class_eval do
+        before_save :slice_attributes_on_changed
+        after_save :clear_changed_attributes
+      end
+    end
+
+    def method_missing(method_symbol, *args)
+      method_name = method_symbol.to_s
+
+      if method_name =~ /(=)$/
+        changed << $`
```

Comment:
I know this is what was used in the source for `ActiveResource` but the intent would be cleaner with a capture group, i.e. `/(.*)(=)$/`